website/docs/sdks/index.md

```diff
@@ -50,56 +50,57 @@ If you see an item marked with a ❌ that you would find useful, feel free to re
+| **Category: [Strategy constraints](../advanced/strategy_constraints)**                            |                        |                           |                    |                            |                        |                           |                      |                                                        |                                   |
+| Basic support (`IN`, `NOT_IN` operators)                                                          | ✅                     | ✅                        | ✅                 | ✅                         | ✅                     | ✅                        | ✅                   | ✅                                                     |                                   |
+| Advanced support (Semver, date, numeric and extended string operators)                            | ✅                     | ✅                        | ⭕                 | ⭕                         | ⭕                     | ⭕                        | ✅                   | ⭕                                                     |                                   |
+| **Category: [Unleash Context](../user_guide/unleash_context)**                                    |                        |                           |                    |                            |                        |                           |                      |                                                        |                                   |
+| Static fields (`environment`, `appName`)                                                          | ✅                     | ✅                        | ✅                 | ✅                         | ✅                     | ✅                        | ✅                   | ✅                                                     |                                   |
+| Defined fields                                                                                    | ✅                     | ✅                        | ✅                 | ✅                         | ✅                     | ✅                        | ✅                   | ✅                                                     |                                   |
+| Custom properties                                                                                 | ✅                     | ✅                        | ✅                 | ✅                         | ✅                     | ✅                        | ✅                   | ✅                                                     |                                   |
+|<span id="current-time-server">`currentTime` context field</span> | ✅                     | ✅                        | ✅                 | ✅                         | ✅                     | ✅                        | ✅                   | ✅                                                     |                                   |
```

Comment:
Which SDKs support the `currentTime` field as of now? I've checked all of them for now, but that may not be the case 💁🏼

Reply:
Everything except .NET, I don't think anyone has tackled this feature in that SDK yet

Reply:
Perfect, thank you!  I'll update that right away.

Reply:
Actually, is this the same as the "Advanced support (Semver, date, numeric and extended string operators)" for strategy contsraints? May be better to just link to that.

Reply:
@sighphyre Got any thoughts on whether they're the same or different?